bgcval2/functions/ice.py

```diff
@@ -54,7 +54,7 @@ def load_area_and_mask(gridfn, maskname,):
     lat = nc.variables['nav_lat'][:]
     nc.close()
     loaded_area_and_mask = True
-    return area, tmask, lat
+    return area.copy(), tmask, lat.copy()
```

Comment:
Ah, but these are global variables. Because I want to load them once and don't want to have to load the mask netcdf file for every data point. Best to treat them carefully.

Reply:
OK - am a bit confused since a global once declared it will stay in memory, but will indeed, get updated to whatever the last instance was called on it - if it works dont fix it then :grin: